### PR TITLE
Visual fixes [SCI-4274]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-4274](https://biosistemika.atlassian.net/browse/SCI-4274)

### What was done
Just few "smaaaaal" visual fixes.

### CODE REVIEW

File: app/views/repositories/show.html.erb [hunk 1]
```diff
@@ -34,15 +34,13 @@
       <% end %>
     </div>
     <div class="repo-datatables-buttons">
-      <div class="new-repository-button">
```
Comment (mlorb) [inline app/views/repositories/show.html.erb]: We need class `new-repository-button` for adding `add_share_inventory_promo_modal` in billing addon.

File: app/assets/javascripts/application.js.erb [hunk 2]
```diff
@@ -191,15 +191,6 @@ var HelperModule = (function(){
 
   var helpers = {};
 
-  helpers.treeLinkTruncation = function() {
```
Comment (mlorb) [inline app/assets/javascripts/application.js.erb]: We still call this function in `app/assets/javascripts/application.js.erb` line 248.

File: app/views/repositories/show.html.erb (hunk 1 above)
Comment (aignatov-bio) [reply app/views/repositories/show.html.erb]: Fix in addon?

**Comment by urbanrotnik** [comment]
@mlorb you can check and merge if everything is good. ;)

**Pull request merged**